crates/polars-python/src/functions/utils.rs

```diff
@@ -1,7 +1,13 @@
 use polars::prelude::_set_check_length;
+use polars_core::config::get_default_engine as _get_default_engine;
 use pyo3::prelude::*;
 
 #[pyfunction]
 pub fn check_length(check: bool) {
     unsafe { _set_check_length(check) }
 }
+
+#[pyfunction]
+pub fn get_default_engine() -> PyResult<Option<String>> {
```

Comment:
I don't think we have to expose this.

Reply:
Responded in https://github.com/pola-rs/polars/pull/20717#discussion_r1943303721